app/src/modules/sparql.js

```diff
@@ -34,8 +39,14 @@ function parseSparql (response) {
         rowData[field] = value
       })
     }
+  } catch (err) {
+    throw new Error('Error parsing response', { cause: err })
```

Comment:
I'll suggest passing the `err` itself as the argument and then removing the object in both cases.